development/docker-compose.keycloak.yml

```diff
@@ -0,0 +1,25 @@
+---
+version: "3.4"
+services:
+  keycloak:
+    image: "quay.io/keycloak/keycloak:15.0.2"
```

Comment:
Do we need to be this specific on the tag, or would something like `keycloak:15` be sufficient?

Reply:
Can take a peak at their versioning. Just copied a known working example for CI